utils/zfp.c

```diff
@@ -534,7 +534,10 @@ int main(int argc, char* argv[])
     }
 
     /* compress data */
-    zfpsize = zfp_compress(zfp, field);
+    for (int i = 0; i < 5; i++) {
```

Comment:
This is useful for profiling but should not be the default behavior in the zfp command-line utility.

Reply:
Thanks for the reminder. That commit was just for profiling. I'm closing this PR for now to avoid confusion.